app/services/CommonTaskListService.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import cats.implicits._
+import jakarta.inject.Singleton
+import models.common.{Mtditid, Nino, TaxYear}
+import models.taskList._
+import play.api.Logging
+import services.journeyAnswers.JourneyStatusService
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class CommonTaskListService @Inject()(journeyStatusService: JourneyStatusService) extends Logging {
+  
+  def get(taxYear: TaxYear, nino: Nino, mtditid: Mtditid)
+         (implicit ec: ExecutionContext, hc: HeaderCarrier): Future[TaskListSection] =
+    journeyStatusService.getCommonTaskList(
```

Comment:
After thinking about this some more, I think it would be worth us having a chat about how this links into the old way of doing things using `JourneyStatusService`. I can see some issues with how that service does things and we can make this a lot simpler.